src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separateParams = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. The variable name `separateParams` could be more descriptive, such as `styleDeclarations` to reflect its content more accurately.

Reply:
Thank you. I will take your wishes into account when performing the following tasks